tests/pages/customer-resource-show.js

```diff
@@ -72,11 +72,35 @@ export default {
   },
 
   get isHidden() {
-    return $('[data-test-eholdings-customer-resource-show-is-hidden]').length === 1;
+    return $('[data-test-eholdings-customer-resource-toggle-hidden] input').prop('checked');
```

Comment:
Yes, this needs to be flipped for `CustomerResourceShowPage.isHidden` to make sense. There might be a few other spots that will need updates to assertions after you make this change.